lib/auth/password_test.go

```diff
@@ -567,6 +567,53 @@ func TestChangeUserAuthentication(t *testing.T) {
 	}
 }
 
+func TestChangeUserAuthentication_DeletePreviousWebSession(t *testing.T) {
+	t.Parallel()
+	srv := newTestTLSServer(t)
+	ctx := context.Background()
+
+	username := "llama@goteleport.com"
+	_, _, err := CreateUserAndRole(srv.Auth(), username, []string{username})
+	require.NoError(t, err)
+
+	// Add some pre web sessions to check for deletion later.
+	sess, err := types.NewWebSession("web-session-id-1", types.KindWebSession, types.WebSessionSpecV2{
```

Comment:
Maybe sess1 so it's all numbered?

```suggestion
	sess1, err := types.NewWebSession("web-session-id-1", types.KindWebSession, types.WebSessionSpecV2{
```